e2etests/cli_test.go

```diff
@@ -0,0 +1,219 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build e2etest
+
+package e2etests
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/DataDog/datadog-traceroute/result"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	cliBinaryPath         string
+	cliBinaryOnce         sync.Once
+	cliBinaryNeedsCleanup bool
+)
+
+// getCLIBinaryPath returns the path to the CLI binary, building it if necessary
+func getCLIBinaryPath(t *testing.T) string {
+	cliBinaryOnce.Do(func() {
+		projectRoot := filepath.Join("..")
+
+		binaryName := "datadog-traceroute"
+		if runtime.GOOS == "windows" {
+			binaryName = "datadog-traceroute.exe"
+		}
+
+		// check for pre-built binary (i.e. when running in CI)
+		preBuiltBinaryPath := filepath.Join(projectRoot, binaryName)
+		if _, err := os.Stat(preBuiltBinaryPath); err == nil {
+			t.Logf("using pre-built binary: %s", binaryName)
+			cliBinaryPath = preBuiltBinaryPath
+			cliBinaryNeedsCleanup = false
+			return
+		}
+
+		testBinaryName := "datadog-traceroute"
```

Comment:
Looks like we can just use `binaryName` name right? `testBinaryName` have the same value